zanata-war/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -196,6 +196,18 @@
     <to last="true">/profile/merge_account.seam$1</to>
   </rule>
 
+  <rule>
+    <from casesensitive="true">^/profile/view/([^/]+)(/([^?]*)\??(.*))?$</from>
```

Comment:
Right now we don't need it. But from Luke's design eventually we will have a similar view with tabs or different portions of the page configured to show different matrices.